addons/point_of_sale/models/pos_order.py

```diff
@@ -933,6 +987,34 @@ def _launch_stock_rule_from_pos_order_lines(self):
                 pickings_to_confirm.action_confirm()
         return True
 
+    def _compute_total_cost(self, stock_moves=None):
+        """
+        Compute the total cost of the order lines.
+        :param stock_moves: recordset of `stock.move`, if present, it means the line is anglo saxon
+        """
+        for line in self:
```

Comment:
maybe we can check that the line has not already be computed (even if it should not happen

Reply:
Why not, I've change this line to `for line in self.filtered(lambda l: not l.is_total_cost_computed):`